src/main/java/ca/team2706/frc/robot/config/Config.java

```diff
@@ -119,6 +119,28 @@ public static void init() {
     public static final FluidConstant<Double> PIGEON_KD = constant("pigeon-ki", 4.0);
     public static final FluidConstant<Double> PIGEON_KF = constant("pigeon-kf", 0.0);
 
+    public static final FluidConstant<String> DRIVER_ASSIST_VISION_CARGO_AND_LOADING = 
```

Comment:
Since these are controller bindings, we would want to see `BINDING` at the end of the variable name so we don't get confused. 

We would also want this in the Networktables string.

Reply:
Changed